pages/price-feeds/error-codes/evm.mdx

```diff
@@ -0,0 +1,21 @@
+# EVM Error Codes
+
+[PythErrors.sol](https://github.com/pyth-network/pyth-crosschain/blob/a888ba318c0325c29070eaf5afcc3a4d443b058c/target_chains/ethereum/sdk/solidity/PythErrors.sol#L8)
+contains the errors that are used in the Pyth Network's EVM contracts.
+
+| Error Codes | Error                          | Error Description                                                        |
+| ----------- | ------------------------------ | ------------------------------------------------------------------------ |
+| 0xa9cb9e0d  | InvalidArgument()              | Function Arguments are invalid.                                          |
```

Comment:
For each of these errors, it would be nice to provide a little bit more information about what it means. These descriptions are in many cases providing no new information beyond the error name. This one is a generic error so may not be possible, but StalePrice for example could say something like "The requested price feed has not been updated recently enough."

Reply:
Agreed. Updated that. We will circle back on this with some detailed troubleshooting in future.